fdbserver/Knobs.cpp

```diff
@@ -616,6 +616,7 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 	//Worker
 	init( WORKER_LOGGING_INTERVAL,                               5.0 );
 	init( HEAP_PROFILER_INTERVAL,                               30.0 );
+	init( JOIN_CLUSTER_WARNING_INTERVAL,                       300.0 );
```

Comment:
I would suggest `REGISTER_WORKER_REQUEST_TIMEOUT`, as it is a timeout for the current request we issued, and we'd be issuing another request.